infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/multiuser/oauth/OpenshiftTokenInitializationFilter.java

```diff
@@ -91,19 +91,11 @@ protected String getUserId(io.fabric8.openshift.api.model.User user) {
   protected Subject extractSubject(String token, io.fabric8.openshift.api.model.User osu) {
     try {
       ObjectMeta userMeta = osu.getMetadata();
-      User user =
-          userManager.getOrCreateUser(
-              getUserId(osu), openshiftUserEmail(userMeta), userMeta.getName());
+      User user = userManager.getOrCreateUser(getUserId(osu), null, userMeta.getName());
```

Comment:
I don't like passing `null` here. At the same time I would not go with some big refactorings. WDYT about overloading `getOrCreateUser` method where new method would not have `email` argument?